porches/generic/site/header.php

```diff
@@ -10,7 +10,18 @@
 ?>
 
 <!-- Required meta tags -->
-<?php echo wp_kses( $porch_fields['google_analytics']['value'] ?: $porch_fields['google_analytics']['default'], $allowed_tags ) ?>
+<?php 
+if ( ! empty( $porch_fields['google_analytics']['value'] ) ) { ?>
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=<?php esc_attr_e( $porch_fields['google_analytics']['value'] ); ?>"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', '<?php esc_attr_e( $porch_fields['google_analytics']['value'] );?>');
```

Comment:
@ansleyglenn 
in `esc_attr_e` the `_e` parts means "translate and echo".
Here we just want to echo, so we use `echo esc_attr()`